src/chains/ethereum/ethereum/src/wallet.ts

```diff
@@ -231,7 +241,8 @@ export default class Wallet {
   };
 
   #initializeAccounts = (
-    options: EthereumInternalOptions["wallet"]
+    options: EthereumInternalOptions["wallet"],
+    logging: EthereumInternalOptions["logging"]
```

Comment:
is `logging` needed here?

Reply:
Yeah that's unnecessary now. I was originally being specific and [doing the check here](https://github.com/trufflesuite/ganache/pull/3361/files/8f787a25df718f8c37a16a91f35125f8398580d5#diff-aec7e0ea6cf2648413b278ca937c3e772944972d94af20a499e1baa07db1a828R285) since I saw the conditionals but the options exclusions made it possible to just do in the constructor.